extensions/dependencies/guppy.html

```diff
@@ -1,8 +1,8 @@
-<link rel="stylesheet" href="/third_party/static/guppy-f6e0bb/build/guppy.min.css">
-<script src="/third_party/static/guppy-f6e0bb/build/guppy.min.js"></script>
+<link rel="stylesheet" href="/third_party/static/guppy-260da66/build/guppy.min.css">
```

Comment:
I thought it was a 6 char hash, but then noticed that Github displayed 7 and copied that. I'll change it to 6 char.